### DISCUSSION TITLE
`code-input.min.js` where is this sourced?

### DISCUSSION THREAD

**Discussion by declann** [adamviola/parquet-explorer#23] [General]
Hi,

This is an awesome extension! Great work.

I'm thinking about the idea to use this to scaffold an extension for [Huey](https://github.com/rpbouman/huey): a pivot table on DuckDB.

I think I've found the key bits of logic but I wonder about `code-input.min.js`, is it a dependency that's been copied or another source file that's been minified?

Thanks a lot

Declan

**Comment by adamviola** [comment]
Yep it's a copied dependency called [code-input](https://github.com/WebCoder49/code-input). Looking forward to seeing Huey as an extension!

**Comment by declann** [comment]
Thanks Adam! That's a really nice component to know
